week-5/DaInglichTim/main.rb

```diff
@@ -0,0 +1,24 @@
+require "csv"
+
+file_product = File.open("queries/product.sql", "w")
+file_customer = File.open("queries/customer.sql", "w")
+file_transaction = File.open("queries/transaction.sql", "w")
+
+product = []
+id_customers = 0
+
+# Lee el csv y retorna cada row
+CSV.foreach("data/sales.csv") do |row|
+  id_customers += 1
+  row[8] = row[8].gsub(/\'/, "`")
+  row[9] = row[9].nil? ? 'NULL' : row[9].gsub(/\'/, "`")
+  row[5] = row[5].gsub(/\'/, "`")
```

Comment:
You could have used two `'` here to escape the `'` and have it as part of the content without issues.